src/main/java/jenkins/branch/MultiBranchProject.java

```diff
@@ -1231,8 +1231,8 @@ private int processHeadCreate(SCMHeadEvent<?> event, TaskListener global, String
                         listener = new LogTaskListener(LOGGER, Level.FINE);
                     }
                     long start = System.currentTimeMillis();
-                    listener.getLogger().format("[%tc] Received %s event with timestamp %tc%n",
-                            start, eventType, eventTimestamp);
+                    listener.getLogger().format("[%tc] Received %s %s event with timestamp %tc%n",
+                            start, eventClass, eventType, eventTimestamp);
```

Comment:
Would help reviewers if the PR summary were to give an example of the expanded log message, relative to the original.